src/GregClient/GregClient.cs

```diff
@@ -85,6 +87,40 @@ public ResponseWithContentBody<T> ExecuteAndDeserializeWithContent<T>(Request m)
 
             return new ResponseWithContent<T>(response).DeserializeWithContent();
         }
+
+        private void logResponse(IRestResponse restResp)
+        {
+            if (restResp == null)
+            {
+                return;
+            }
+            try
+            {
+                var logDirPath = Path.Combine(Path.GetTempPath(), "GregLog");
+                if (!Directory.Exists(logDirPath))
+                {
+                    Directory.CreateDirectory(logDirPath);
+                }
+                using (StreamWriter outputFile = new StreamWriter(Path.Combine(logDirPath, Guid.NewGuid().ToString() + ".txt")))
```

Comment:
Should be a timestamp instead of a guid, for easier aligning with requests?

Reply:
ok..might be more readable

Reply:
My first thought was to try a config file for Greg.dll
`Greg.dll.config`:
```
<?xml version="1.0" encoding="utf-8" ?>
<configuration>
  <appSettings>
    <add key="debugLog" value="true" />
  </appSettings>
</configuration>
```
But I am not sure yet how to load such a config file...Trying to search for the MSDN  api now...
Any other ideas ? not sure how to make `env vars` work in this case...

Reply:
Oh..I was thinking of process environment variables...
I am guessing you meant system wide env vars..that might work too